zipkin-lens/webpack.prod.config.js

```diff
@@ -40,24 +46,12 @@ module.exports = {
       },
       {
         test: /\.scss$/,
-        use: ExtractTextPlugin.extract({
-          fallback: 'style-loader',
-          use: [
-            {
-              loader: 'css-loader',
-              options: {
-                url: false,
```

Comment:
Not sure why we had `url: false`, but we definitely need `url: true` to get the font paths into the Webpack pipeline (both for URL rewriting, and to actually output the font files into the target directory)